LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 James Lahey
```

Comment:
Let's change `James Lahey` to `Root Inc.`

Reply:
Fixed in latest push